ambry-router/src/main/java/com.github.ambry.router/GetOperation.java

```diff
@@ -129,12 +129,6 @@ boolean isOperationComplete() {
   }
 
   /**
-   * Return the {@link MessageFormatFlags} to associate with the requests for this operation.
-   * @return the {@link MessageFormatFlags} to associate with the requests for this operation.
-   */
-  abstract MessageFormatFlags getOperationFlag();
```

Comment:
Previously, I was planning on using a getBlobAll call for the first chunk, in which case the operation flag would be chunk-specific.  However, now that we aren't doing that I can make this a GetOperation method again.